lib/studies/reply/adaptive_nav.dart

```diff
@@ -240,46 +203,40 @@ class _DesktopNavState extends State<_DesktopNav>
                       minHeight: constraints.maxHeight,
                     ),
                     child: IntrinsicHeight(
-                      child: NavigationRail(
-                        destinations: [
-                          for (var destination in widget.destinations)
-                            NavigationRailDestination(
-                              icon: Material(
-                                color: Colors.transparent,
-                                child: ImageIcon(
-                                  AssetImage(
-                                    destination.icon,
-                                    package: _assetsPackage,
+                      child: ValueListenableBuilder<bool>(
+                        valueListenable: _isExtended,
+                        builder: (context, value, child) {
+                          return NavigationRail(
+                            destinations: [
+                              for (var destination in widget.destinations)
+                                NavigationRailDestination(
+                                  icon: Material(
+                                    color: Colors.transparent,
+                                    child: ImageIcon(
+                                      AssetImage(
+                                        destination.icon,
+                                        package: _assetsPackage,
+                                      ),
+                                    ),
                                   ),
+                                  label: Text(destination.name),
                                 ),
-                              ),
-                              label: Text(destination.name),
+                            ],
+                            extended: _isExtended.value,
+                            labelType: NavigationRailLabelType.none,
+                            leading: _NavigationRailHeader(
+                              extended: _isExtended,
```

Comment:
Nit: can `_NavigationRailHeader` fit on one line?

Reply:
It does not fit